common/flatpak-dir.c

```diff
@@ -7218,6 +7292,10 @@ flatpak_dir_deploy_update (FlatpakDir   *self,
 
   flatpak_dir_cleanup_removed (self, cancellable, NULL);
 
+  commit = flatpak_dir_read_active (self, ref, cancellable);
+  flatpak_dir_log (self, "deploy update", old_origin, ref, commit, NULL,
+                   "Updated %s from %s", ref, old_origin);
```

Comment:
I wonder if this should log the old deploy commit too. I realize this is duplicated if you have an infinite log, but that may not always be true, and even then looking up the previous version might be somewhat of a pain if it was a long time ago.

Reply:
sure, can do